core-js/pom.xml

```diff
@@ -41,41 +41,58 @@
       <groupId>org.webjars.npm</groupId>
       <artifactId>fancyapps__fancybox</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.webjars.npm</groupId>
+      <artifactId>underscore</artifactId>
+    </dependency>
   </dependencies>
 
   <build>
     <plugins>
-      <!-- copy jquery for tests and lib-->
       <plugin>
-        <groupId>com.coderplus.maven.plugins</groupId>
-        <artifactId>copy-rename-maven-plugin</artifactId>
-        <version>1.0</version>
+        <artifactId>maven-resources-plugin</artifactId>
```

Comment:
Some of the copy/rename dependencies work previously done in the `pom.xml` was moved to the `assembly.xml` file.

These new maven plugin executions deal with making sure the assembly result is available for both production (using the `zip` format) and testing (using the `dir` format).